api/openapi.gen.json

```diff
@@ -305,21 +370,8 @@
           "body": {
             "type": "string"
           },
-          "fingerprint": {
-            "type": "string"
-          },
-          "fingerprint_legacy": {
-            "type": "string"
-          },
-          "id": {
-            "format": "int64",
-            "type": "integer"
-          },
```

Comment:
This doesn't sound correct :)

Reply:
It is correct actually, these are all fields generated by the server, so request payload must not contain it. Previously we did not care so it was all present (and we actually enforced an empty string on fingerprints for security reasons). This patch correctly removes this from the request payload.